backend/src/Designer/Infrastructure/GitRepository/AltinnAppGitRepository.cs

```diff
@@ -709,6 +715,41 @@ public string[] GetOptionListIds()
             return optionListIds.ToArray();
         }
 
+        /// <summary>
+        /// Saves bpmn file to disk.
+        /// </summary>
+        /// <param name="file">Content of bpmn file.</param>
+        /// <returns></returns>
+        /// <exception cref="ArgumentException">Throws argument exception if size of syntax validation of the file fails.</exception>
+        public async Task<string> SaveProcessDefinitionFile(string file)
+        {
+            Guard.AssertNotNullOrEmpty(file, nameof(file));
+            if (file.Length > 100_000)
```

Comment:
Why do we need this check?

Reply:
To prevent huge data attacks. We're loading xml in a parser as a check if it's a valid xml file.
It's a small check to prevent our memory to go wild. 
I'm pretty sure our bpmn file shouldn't be longer than that.